LEAF_Request_Portal/FormWorkflow.php

```diff
@@ -117,10 +117,9 @@ public function getCurrentSteps(): array|int|null
                     $quadGroupIDs = $this->login->getQuadradGroupID();
                     $vars3 = array(
                         ':serviceID' => $res[$i]['serviceID'],
-                        ':quadGroupIDs' => $quadGroupIDs
```

Comment:
I feel like this should stay.

Reply:
I mentioned this to Michael Gao because I wanted to confirm this as well.  QuadIds is coming directly from us, and the IDs composing it from the db are enforced int types, so this should be a safe query.  There is also some additional complexity that would be associated with using parameterized variables inside WHERE ... IN

Reply:
the original change had "WHERE groupID IN (:quadGroupIDs)" which did not work. It seemed like the data was getting escaped as a string since single ids would work but multiple did not. IE: WHERE groupID IN ('100,101') vs WHERE groupID IN (100,101). That is how it was rendering in my head.